### ISSUE TITLE
Incorrect resistor part number in BOM

### ISSUE THREAD

**Issue by FrerichsJacob** [StuckAtPrototype/Racer#39]
Racer and Thumbtroller BOMs list RMCF0603JT5R10 (5.1 ohm) where they should say RMCF0603JT5K10 (5.1k ohm)

**Comment by StuckAtPrototype** [comment]
Agreed, thats a mistake in the part number. The entire BOM should be double checked
